core/src/main/java/hudson/ClassicPluginStrategy.java

```diff
@@ -75,6 +75,7 @@
 import org.apache.tools.zip.ZipExtraField;
 import org.apache.tools.zip.ZipOutputStream;
 
+@SuppressFBWarnings(value = "DP_CREATE_CLASSLOADER_INSIDE_DO_PRIVILEGED", justification = "Jenkins is not designed to work with a security manager")
```

Comment:
Can we just add this to the exclude list pending https://github.com/spotbugs/spotbugs/issues/1515?

Reply:
Done